dlc_developer_config.toml

```diff
@@ -23,7 +23,7 @@ benchmark_mode = false
 [build]
 # Frameworks for which you want to disable both builds and tests
 # available frameworks - ["autogluon", "huggingface_tensorflow", "huggingface_pytorch", "huggingface_tensorflow_trcomp", "huggingface_pytorch_trcomp", "tensorflow", "mxnet", "pytorch"]
-skip_frameworks = []
+skip_frameworks = ["tensorflow", "huggingface_tensorflow", "huggingface_tensorflow_trcomp", "mxnet", "autogluon"]
```

Comment:
Can we edit this and skip everything except "pytorch"

Reply:
Shdn't we keep all the pytorch ones i.e "huggingface_pytorch", "huggingface_pytorch_trcomp", "pytorch"

Reply:
No, we need to remove those as we are focusing on PT 1.11 only